src/cr/cube/matrix/cubemeasure.py

```diff
@@ -87,851 +94,1251 @@ def _slice_idx_expr(cls, cube, slice_idx):
         return np.s_[slice_idx]
 
 
-# === MEANS ===
+# === COUNTS (UNWEIGHTED & WEIGHTED) ===
```

Comment:
Note that Unweighted counts have not yet been migrated yet, I realized I should have been asking for more feedback sooner, so I split this PR off.